src/components/Acordition/index.tsx

```diff
@@ -0,0 +1,33 @@
+import type {ReactNode} from 'react';
+import React from 'react';
+import {View} from 'react-native';
+import type {SharedValue} from 'react-native-reanimated';
+import Animated, {useAnimatedStyle, useDerivedValue, useSharedValue, withTiming} from 'react-native-reanimated';
+
+function AccordionItem({isExpanded, children, duration = 300}: {isExpanded: SharedValue<boolean>; children: ReactNode; duration?: number}) {
```

Comment:
Please make sure the file and main export are named the same:
Component is called `AccordionItem` but the file is `Accordion/index.ts` (you have a type in the name).